nautobot/docs/docker/index.md

```diff
@@ -4,37 +4,46 @@ Nautobot is packaged as a Docker image for use in a production environment. The
 
 ## Tags
 
-The Docker image is published to Docker Hub.
+A set of Docker images are built for each Nautobot release and published to both [Docker Hub](https://hub.docker.com/r/networktocode/nautobot/) and [GitHub Packages](https://github.com/nautobot/nautobot/pkgs/container/nautobot).
 
-To get the image from Docker Hub run:
+Additionally, GitHub Actions are used to automatically build images corresponding to each commit to the `develop` and `next` branches; these images are only published to GitHub Packages.
+
+To get a specific tagged image from Docker Hub or GitHub Packages run:
 
 ```no-highlight
-docker image pull networktocode/nautobot
+docker image pull networktocode/nautobot:${TAG}
```

Comment:
You can only get tagged images for GitHub Packages... It might be more accurate to say "the GitHub Container Registry" here and above.  Also the docker command would be:

```no-highlight
docker pull ghcr.io/nautobot/nautobot:${BRANCH}-${SHORT_TAG}-${TIMESTAMP}-py${PYTHON_VERSION}
```

Reply:
I was using `${TAG}` as a generic variable (for example "1.1.6" or "stable" would be valid values for TAG), not a reference to a commit hash. Is there a different variable name that I could use that would be clearer here?

Reply:
@nniehoff Does that make sense?

Reply:
You might add a note to see the table below for valid tags or something.  When we first wrote this I was given direction to be explicit in the docs vs the generic case, however, I agree if people are reading this doc they probably understand what $TAG is.

Reply:
Understood. I think the current text kinda states this?

![image](https://user-images.githubusercontent.com/5603551/146023943-21969c70-0cec-4929-850a-15baa7ca6e44.png)

Reply:
agreed, yes this looks great, sorry should have looked at the rendered markdown vs the diff hard to see through the weeds

Reply:
No worries!